static/app/views/settings/organizationIntegrations/addIntegrationButton.tsx

```diff
@@ -51,7 +52,16 @@ export function AddIntegrationButton({
           <Button
             disabled={!provider.canAdd}
             {...buttonProps}
-            onClick={() => onClick()}
+            onClick={() => {
+              if (label === t('Reinstall')) {
+                trackAnalytics('integrations.integration_reinstall_clicked', {
+                  organization,
+                  organization_id: organization.id,
```

Comment:
```suggestion
                  organization,
```
organization is enough